wrappers/s2i/python/Dockerfile.gpu

```diff
@@ -2,11 +2,28 @@ FROM nvidia/cuda:10.0-cudnn7-runtime-ubuntu18.04
 
 LABEL io.openshift.s2i.scripts-url="image:///s2i/bin"
 
-# Install Python 3 and symlink `pip` and `python` to use Python 3
-RUN apt-get update -y && \
-    apt-get install -y python3-pip python3-dev && \
-    ln -s /usr/bin/pip3 /usr/bin/pip && \
-    ln -s /usr/bin/python3 /usr/bin/python
+RUN apt-get update -y && apt-get install -qy --no-install-recommends \
+    wget \
+    libsm6 \
+    libxext6 \
+    libxrender-dev \
+    libglib2.0-0 \
+    && \
+    rm -rf /var/lib/apt/lists/*
+
+ENV PATH /opt/conda/bin:$PATH
+
+# Install Miniconda3
+RUN wget --quiet https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh -O ~/miniconda.sh && \
```

Comment:
May be a good idea to fetch fixed version and check control sum.

Reply:
I'm referring to continuumio/miniconda3 source code from https://github.com/ContinuumIO/docker-images/blob/master/miniconda3/debian/Dockerfile, I believe use Miniconda3-latest-Linux-x86_64.sh without lockdown the version is ok because the major version will always be 3 and it should be backward compatible?

Reply:
`3` there would correspond to Python version. conda itself is now on `4` sth.
My point is more about making it more reproducible - so we know what is installed.

Reply:
thanks for point that out, I agree with you lockdown conda version is a good idea, updated

Reply:
Nice, much better. If you could make it use version defined in the `Makefile`, 4.7.12, like other images do it would be even better! :)

Reply:
I mean, use `ARG CONDA_VERSION` and then `$CONDA_VERSION`.

Reply:
updated